src/node.cc

```diff
@@ -759,9 +764,9 @@ Local<Value> ErrnoException(Isolate* isolate,
         String::Concat(cons3, String::NewFromUtf8(env->isolate(), path));
     Local<String> cons5 =
         String::Concat(cons4, FIXED_ONE_BYTE_STRING(env->isolate(), "'"));
-    e = Exception::Error(cons5);
+    e = v8::Exception::Error(cons5);
```

Comment:
XLC apparently has trouble finding "Exception::Error" even though we're "using namespace v8", resulting in compile errors. It was necessary to explicitly use the v8 namespace to resolve the errors.